.github/workflows/release-stable.yml

```diff
@@ -54,33 +26,55 @@ jobs:
           distribution: 'zulu'
           java-package: 'jdk+fx'
 
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
-
       - name: Gradle build
         uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
         with:
           arguments: build
 
+      - name: Remove Snapshot Suffix
+        id: removeSnap
+        uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
+        with:
+          arguments: removeSnapshotSuffix
+
+      - name: Remove Snapshot Suffix
+        run: |
+          git config --global user.name 'acrolinx-machine-user-for-integrations'
+          git config --global user.email 'danjela.lura@acrolinx.net'
+          git commit -m "Release v${{ steps.removeSnap.TAGNAME }}"
+
       - name: Create Javadoc
         uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
         with:
           arguments: javadoc
 
       - name: Deploy artifacts
-        id: deploy
         if: github.ref == 'refs/heads/main'
-        run: ./deploy.sh
+        uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
+        with:
+          arguments: publishToSonatype closeAndReleaseStagingRepository --stacktrace
 
       - name: Bump version and push tag
```

Comment:
What version is being "bumped" here? I think this action only creates and pushes a Git tag?
Btw, if we already use raw git commands, maybe we could also run a simple `git tag ${{ steps.removeSnap.TAGNAME }}` to create the tag and a `git push origin ${{ steps.removeSnap.TAGNAME }}` to push it? Then we don't need this action here at all?